pkg/cloudprovider/provider/gce/types/types.go

```diff
@@ -51,6 +51,7 @@ type CloudProviderSpec struct {
 	EnableNestedVirtualization   providerconfigtypes.ConfigVarBool    `json:"enableNestedVirtualization,omitempty"`
 	MinCPUPlatform               providerconfigtypes.ConfigVarString  `json:"minCPUPlatform,omitempty"`
 	GuestOSFeatures              []string                             `json:"guestOSFeatures,omitempty"`
+	ProjectID                    providerconfigtypes.ConfigVarString  `json:"projectID,omitempty"`
```

Comment:
We should add this field to the example manifest: https://github.com/kubermatic/machine-controller/blob/main/examples/gce-machinedeployment.yaml

Reply:
Done